pmda/rmsf.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+import MDAnalysis as mda
+import pmda
```

Comment:
Remove this line.

Reply:
The imports are probably correct in `rms.py`.